app/controllers/api/v1/sessions_controller.rb

```diff
@@ -32,13 +32,18 @@ def index
       # POST /api/v1/sessions
       # Signs a user in and updates the session cookie
       def create
-        user = User.find_by(email: session_params[:email], provider: current_provider)
-
         # TODO: Add proper error logging for non-verified token hcaptcha
         return render_error if hcaptcha_enabled? && !verify_hcaptcha(response: params[:token])
 
+        user = User.find_by(email: session_params[:email])
```

Comment:
It is assumed that an email won't be duplicated in the database

Reply:
It is definitely possible that there will be the same email available in separate providers. This breaks that functionality